clients/src/main/java/org/apache/kafka/common/metrics/stats/Min.java

```diff
@@ -22,7 +22,7 @@
 public class Min extends SampledStat {
 
     public Min() {
-        super(Double.MIN_VALUE);
+        super(Double.MAX_VALUE);
```

Comment:
Nice catch (and sorry for long delay in review).
I saw we are using NEGATIVE_INFINITY for Max() default. Any reason not to use INFINITY here? It seems more consistent.